my-react-app/src/redux/store.js

```diff
@@ -1,5 +1,11 @@
-import { createStore } from 'redux';
+import { createStore, combineReducers } from 'redux';
+import { reducer as formReducer } from 'redux-form';
 
-import Reducer from './game/reducer';
```

Comment:
Keep the game reducer.
`import reducer as gameReducer from './game/reducer';`
the add gameReducer to the `reducers` object